code/agent/config/dev_objects.json

```diff
@@ -105,7 +160,7 @@
             "z": 1.60,
             "roll": 0.0,
             "pitch": 0.0,
-            "yaw": 0.0
+            "yaw": -45.0
```

Comment:
I guess that was for testing purposes, so you can test the lidar? When testing with stuff from planning i figuered that this causes the car to turn left all the time. please remove before merging